### ISSUE TITLE
Influxdb2, data formatting and plugin configuration

### ISSUE THREAD

**Issue by maxdd** [panodata/grafana-map-panel#96]
Hi,
i noticed this plugin after struggling with worldmap-panel.
I admit there is a little bit of confusion between the two in the sense that i have no idea what are the differences nor which of this is maintained.

In any case i have a influxdb2 setup with grafana and i'm using a python script with influxdb-client which create a point

```
p = Point(measurement_name).tag("key", ISO) \
		.tag("latitude", Lat) \
		.tag("longitude", Long) \
		.tag("Domain", Domain) \
		.tag("City", City) \
		.tag("State", State) \
		.tag("Country", Country) \
		.tag("IP", IP) \
		.field("Domain", Domain) \
		.field("latitude", Lat) \
		.field("longitude", Long) \
		.field("City", City) \
		.field("State", State) \
		.field("key", ISO) \
		.field("name", Country) \
		.field("IP", IP) \
		.field("duration", duration) \
		.field("metric", 1) 
```

I've tried different combinations of queries e.g.

```
from(bucket: "npm-graf")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r["_measurement"] == "ReverseProxyConnections" and r["_field"] == "IP")
  |> group(columns: ["IP", "latitude", "longitude"], mode:"by")
  |> count(column: "_value")
```

https://user-images.githubusercontent.com/6155973/113273001-7ea2f000-92dc-11eb-9b22-0ba8f70f2e8e.png

but i have yet to find a way to make it work (i've just started with grafana).
Long story short is there/can you provide a clear example that shows how the timeseries/table should be and how the panel shall be configured?

Regards,

**Comment by maxdd** [comment]
Any help?

**Comment by amotl** [comment]
Hi Max,

apologies for the very late reply. Personally, I never operated our fork with either InfluxDB2 nor with a Flux query.

But I believe it should work with Flux in general. While I am also not able to currently point out a clear configuration example for that environment, chances are that @wetterfrosch might be able to do so.

If you don't hear back from anyone, please ping me again around/after 15th of July, when I will be back from vacation. Then, I will try to find some time to outline a concise and working example for your specific environment.

Thanks for your patience.

With kind regards,
Andreas.

On 1 July 2021 14:41:22 CEST, maxdd ***@***.***> wrote:
>Any help?
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/panodata/grafana-map-panel/issues/96#issuecomment-872213056

-- 
Sent from my mind. This might have been typed on a mobile device, so please excuse my brevity.

**Comment by amotl** [comment]
Oh, and to work against any confusion: The page at https://community.panodata.org/t/grafana-map-panel/121 outlines the most important updates and additional features compared to the original Worldmap Panel.

Cheers!

On 1 July 2021 14:41:22 CEST, maxdd ***@***.***> wrote:
>Any help?
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/panodata/grafana-map-panel/issues/96#issuecomment-872213056

-- 
Sent from my mind. This might have been typed on a mobile device, so please excuse my brevity.

**Comment by maxdd** [comment]
@amotl i guess it's time to ping you back :D

**Comment by maxdd** [comment]
@amotl any feedback on this?
